practice/ttt/user_input.clj

```diff
@@ -0,0 +1,30 @@
+(ns ttt.user-input
```

Comment:
suggestion/question: What do you think about naming this layer `cli` instead of `user-input`? The functions here are very specific to parsing user input _from the command line_. We might have an entirely different set of validations for user input if we were to create a web UI interface for the game, and so this namespace might not entirely make sense at that point. It's a stretch of course (since we don't plan on building a web UI), but still something to think about.

Reply:
I understand, the user-input namespace is never reading directly from the cli. This particular namespace has no idea that a cli exists, it's just parsing input that is not necessarily from the command line. Thats why I think naming it user-input works better.